app.py

```diff
@@ -9,7 +9,7 @@
 BASE_URL = 'https://api.nytimes.com/svc/search/v2/articlesearch.json'
 
 params = {
-    'q': 'braves', # Query keywords
+    'q': 'braves', # Query keywords  
```

Comment:
I add two spaces to this line because I accidentally pushed my code to GitHub too early and I needed to make a change in order to commit.